### PR TITLE
First pass at JSF

### PR DESCRIPTION
**Pull request by ggam**

### CODE REVIEW

**Comment by ggam** [comment]
AFAIK, Arjan plans to request a name change to Jakarta Faces so you are right this document is currently not ok, but for the time it gets published, will probably be fine. @arjantijms can you please confirm?

**Comment by ggam** [comment]
I have renamed to Jakarta Server Faces, fixed all underlines I saw and replaced some JCP links with the new jakarta.ee ones. I also added a task to #147 for revewing all other Jakarta Faces references on the remaining files.

File: src/main/jbake/content/jsf-ajax012.adoc [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 
-For more information on Ajax in JavaServer Faces Technology, see
+For more information on Ajax in Jakarta Server Faces Technology, see
 
-* JavaServer Faces project website:
+* Jakarta Server Faces project website:
 +
 `https://javaserverfaces.github.io/users.html`
-* link:../javaserver-faces-2-2/jsdocs/symbols/jsf.ajax.html[JavaServer
+* link:../javaserver-faces-2-2/jsdocs/symbols/jsf.ajax.html[Jakarta Server
```
Comment (bshannon) [inline src/main/jbake/content/jsf-ajax012.adoc]: I think this should be a link to https://jakarta.ee/specifications/faces/2.3/jsdoc.

File: src/main/jbake/content/jsf-custom003.adoc [hunk 2]
```diff
@@ -30,10 +30,10 @@ Map?]
 
 [[GLPBD]][[why-use-javaserver-faces-technology-to-implement-an-image-map]]
 
-Why Use JavaServer Faces Technology to Implement an Image Map?
+Why Use Jakarta Server Faces Technology to Implement an Image Map?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (pzygielo) [inline src/main/jbake/content/jsf-custom003.adoc]: += ~~~~

File: src/main/jbake/content/jsf-develop001.adoc [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 ~~~~~~
-= Managed Beans in JavaServer Faces Technology
+= Managed Beans in Jakarta Server Faces Technology
 
 
 [[BNAQM]][[managed-beans-in-javaserver-faces-technology]]
 
-Managed Beans in JavaServer Faces Technology
+Managed Beans in Jakarta Server Faces Technology
 --------------------------------------------
```
Comment (pzygielo) [inline src/main/jbake/content/jsf-develop001.adoc]: += ----

**Pull request merged**